pkg/ai/iai.go

```diff
@@ -5,12 +5,17 @@ import (
 )
 
 type IAI interface {
-	Configure(token string, model string, language string) error
+	Configure(config IAIConfig, language string) error
 	GetCompletion(ctx context.Context, prompt string) (string, error)
 	Parse(ctx context.Context, prompt []string, nocache bool) (string, error)
 	GetName() string
 }
 
+type IAIConfig interface {
+	GetToken() string
```

Comment:
To be more generic, shouldn't rename this to `GetPassword` ?

Reply:
fair !